rust/ql/lib/codeql/rust/elements/internal/CastExprImpl.qll

```diff
@@ -12,11 +11,16 @@ private import codeql.rust.elements.internal.generated.CastExpr
  * be referenced directly.
  */
 module Impl {
+  // the following QLdoc is generated: if you need to edit it, do it in the schema file
   /**
    * A cast expression. For example:
    * ```rust
    * value as u64;
    * ```
    */
-  class CastExpr extends Generated::CastExpr { }
+  class CastExpr extends Generated::CastExpr {
+    override string toString() {
+      result = this.getExpr().toAbbreviatedString() + " as " + this.getTy().toString()
```

Comment:
We should avoid, if possible, to make `toString` recursive. Could we instead have a predicate, `getName`, on `TypeRef`?